services/slam/server_test.go

```diff
@@ -75,7 +105,8 @@ func TestWorkingServer(t *testing.T) {
 	pcd, err := os.ReadFile(cloudPath)
 	test.That(t, err, test.ShouldBeNil)
 
-	t.Run("working get position function", func(t *testing.T) {
+	//nolint:dupl
```

Comment:
all the `//nolint:dupl` will be removed once the temp grpc methods are removed.

Reply:
For my understanding, what is the linter detecting here? (I'm wondering since the test case names ("working get position new function" and "working GetPosition") and test case bodies are both different)